sunkit_image/coalignment/__init__.py

```diff
@@ -0,0 +1,4 @@
+from sunkit_image.coalignment.interface import coalignment
+from sunkit_image.coalignment.match_template import match_template_coalign  # noqa : F401
+
+__all__ = ["coalignment"]
```

Comment:
The top level needs to expose the decorators, the `registered_methods` dict and the named tuple.